client/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:20-alpine as builder
+WORKDIR /app
+COPY ./package.json package-lock.json ./
+RUN npm install
+COPY . .
+RUN npm run build
+
+FROM nginx:stable-alpine
```

Comment:
shouldn't we use specified version on nginx instead of latest (idk what version it is)?

Reply:
nope hehe